File: FSA Claim/ClientApp/src/app/services/data-classes/GetClaimsResult.ts
import { IGetClaimResult } from "../interfaces/IGetClaimsResult";
import { IViewClaim } from "../interfaces/IViewClaim";

export class GetClaimsResult implements IGetClaimResult {
  claims: IViewClaim[] = [];
  fsaAmount: number = 0;
  yearCoverage: number = 0;
  employeeID: number = 0;
  employeeName: string = "";
  availableFSA: number = 0;
  pendingClaims: number = 0;
  approvedClaims: number = 0;

}

File: FSA.API/Models/GetClaimsResult.cs
using FSA.API.Models.Interface;

namespace FSA.API.Models
{
    public class GetClaimsResult : IGetClaimsResult
    {
        public IEnumerable<IViewClaim> Claims { get; set; }
        public decimal FSAAmount { get; set; }
        public int YearCoverage { get; set; }
        public int EmployeeID { get; set; }
        public string EmployeeName { get; set; }
        public decimal AvailableFSA {get; set; }
        public decimal PendingClaims { get; set; }
        public decimal ApprovedClaims { get; set; }
    }
}
